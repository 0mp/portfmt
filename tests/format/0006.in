CMAKE_ARGS=	-D foo -D bla
CMAKE_ARGS+=	-D x -D y
