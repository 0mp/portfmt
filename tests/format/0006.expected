CMAKE_ARGS=	-Dbla \
		-Dfoo \
		-Dx \
		-Dy
